wordpress-mobile/WordPress-Android
# Domain transfer dashboard card

Adds dashboard card for domain transfer

Fixes: [See](pc5zdG-1b2-p2) 

| Domain Transfer Card | Domain Transfer Intro |
|--------|--------|
| ![Screenshot_20230802_133138](https://github.com/wordpress-mobile/WordPress-Android/assets/990349/fde5e739-9a94-4319-bae7-c7cb308e2e1e) |![Screenshot_20230729_145335](https://github.com/wordpress-mobile/WordPress-Android/assets/990349/026d3e4a-33f5-4db3-8c76-776c930dbef8) | 



To test:

Launch Jetpack app
`Verify` **domains transfer card** appears as shown above on dashboard
`Tap` on the Learn more or anywhere on the card
`Verify` it launches [domain transfer page](https://wordpress.com/setup/domain-transfer/intro)
`Tap` on overflow (three dots) menu on the card
`Verify` it pops up menu with **Hide this** option
`Tap` on Hide this
`Verify`, the card gets hidden 

Check for the following tracks in AS
🔵 Tracked: dashboard_card_domain_transfer_shown
🔵 Tracked: dashboard_card_domain_transfer_more_menu_tapped
🔵 Tracked: dashboard_card_domain_transfer_hidden


>**Note**
> Copy on the card could change later

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Unit tests and manual tests

3. What automated tests I added (or what prevented me from doing so)
Updated and added unit tests

PR submission checklist:



UI Changes testing checklist:
